tasks/app1-2.js

```diff
@@ -0,0 +1,25 @@
+const csv=require("csvtojson/v2");
+const csvFilePath='./csv/nodejs-hw1-ex1.csv';
+const outputTxtFile='./output/new_file2.txt';
+const fs = require('fs');
+const { pipeline } = require('stream');
+
+pipeline(
+  csv({
+    colParser:{
+      "amount": "omit",
+  		"price":"number",
+  	},
+    headers: ['book','author','amount','price']
+
+  }).fromStream(fs.createReadStream(csvFilePath)),
```

Comment:
Consider creating a read stream before the csv logic in the pipeline.
This will help you to get rid of "fromStream" call since the "csv" job will be applied to created stream.